lib/daterangepicker/daterangepicker.js

```diff
@@ -250,7 +250,11 @@
             }
         },
 
-        showMonth: function(){
+        showMonth: function(selectStartDate){
+            if(selectStartDate){
```

Comment:
I guess you would want a test to cover this branch? Furthermore, does overwriting the "state" of the `daterangepicker` here have any impact on things later?